api-tests/dev_apis/crypto/test_c058/test_c058.c

```diff
@@ -139,7 +139,7 @@ int32_t psa_aead_update_test(caller_security_t caller __UNUSED)
             TEST_ASSERT_EQUAL(status, PSA_ERROR_BAD_STATE, TEST_CHECKPOINT_NUM(9));
         } else {
             /* Compare the output and its length with the expected values */
-            TEST_ASSERT_RANGE(output_length, 0, check1[i].expected_output_length,
```

Comment:
If I remember correctly, the PSA spec for a call to psa_aead_update() does not mandate to return an exact value, but it can return anything from 0 up to the expected_output_length in this case. So probably the best solution here is not to change the macro but just cast the output_length, 0 and check1[i].expected_output_length to (int32_t) to avoid the compiler warnings (that in the TF-M build result in a build failure).

Reply:
Agreed! Please help to check if I rebased successfull~~